mava/configs/env/cleaner.yaml

```diff
@@ -0,0 +1,18 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+  - scenario: clean-5x5x3a # [clean-10x10x10a, clean-15x15x23a, clean-20x20x40a]
+# Further environment config details in "clean-5x5x3a" file.
+
+env_name: Cleaner # Used for logging purposes.
+
+# Defines the metric that will be used to evaluate the performance of the agent.
+# This metric is returned at the end of an experiment and can be used for hyperparameter tuning.
+eval_metric: episode_return
+
+# Whether the environment observations encode implicit agent IDs. If True, the AgentID wrapper is not used.
+# This should not be changed.
+implicit_agent_id: True
+
+kwargs:
+  time_limit: 100
```

Comment:
I worry about this being an argument of the env. Although as long as the reward encorages agents to finish as soon as possible time limit shouldn't matter right?

Reply:
I agree the time limit should be a scenario argument. But Ill make a separate issue for this since it involves changing all the other envs too.